docs/guidelines/index.md

```diff
@@ -10,4 +10,7 @@ This section contains guidelines relevant anyone writing code in Equinor.
 - [Snyk](../snyk/index.md)
 - [Scanning for Secrets in code](secret-scanning.md)
 - [Postman](postman.md)
-- [Authentication and Authorization](authn-authz.md)
\ No newline at end of file
+- [Authentication and Authorization](authn-authz.md)
+
+!!! question "Ask us on Slack"
```

Comment:
Should we add information that this is Equinor Interal, like  "Ask us on Slack (Equinor Internal only)"?